owncloudApp/src/main/res/layout/account_item.xml

```diff
@@ -68,33 +68,18 @@
 
     <TextView
         android:id="@+id/account"
-        android:layout_width="210dp"
+        android:layout_width="0dp"
         android:layout_height="44dp"
         android:layout_marginStart="8dp"
         android:ellipsize="end"
         android:text="@string/placeholder_sentence"
         android:textColor="@color/textColor"
         android:textSize="14sp"
+        android:lines="1"
```

Comment:
I would add as well an ellipsizing option, in case the text is longer than the available space

Reply:
Do you mean the `ellipsize`? It is incorporated

Reply:
Yes, but change the value to `middle`